### PR TITLE
convert check for understandings into questions (Part C)

### PR DESCRIPTION
**Pull request by Kamilahsantos**
In this PR I convert check for understandings into questions, of issue #1005

### CODE REVIEW

File: giving-presentations/giving-presentations.md [hunk 1]
```diff
@@ -114,4 +114,9 @@ Try crafting a 5-minute technical presentation about a homework assignment or pr
 
 ### Check for Understanding
 
-Practice makes perfect: have students write and deliver speeches, and practice giving and receiving feedback on their performance.
+- How important is it to make good presentations for your career?
+- How to create a good outline?
```
Comment (alodahl) [inline giving-presentations/giving-presentations.md]:
```suggestion
- How do you create a good outline?
```

File: giving-presentations/giving-presentations.md [hunk 2]
```diff
@@ -114,4 +114,9 @@ Try crafting a 5-minute technical presentation about a homework assignment or pr
...
+- Important points for creating good slides?
```
Comment (alodahl) [inline giving-presentations/giving-presentations.md]:
```suggestion
- What are Important things to remember for creating good slides?
```

Comment (Kamilahsantos) [reply giving-presentations/giving-presentations.md]: Thanks for the review, I saved the suggestions

**Comment by alodahl** [comment]
fixes #1005

**Pull request merged**